File: Airport/Shared/DTO/Aircraft.cs
using System;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;

namespace Shared.DTO
{
    public class Aircraft
    {
       
        public int Id { get; set; }
        [Required,StringLength(100)]
        public string AircraftName { get; set; }
        [Required]
        public AircraftType AircraftType { get; set; }
        [Required]
        public DateTime AircraftReleaseDate { get; set; }
        [Required]
        public TimeSpan ExploitationTimeSpan { get; set; }
        
    }
}

File: angular/src/app/Shared/Models/aircraft.ts
import { AircraftType } from './aircraftType';

export class Aircraft {
    constructor(
        public id?: number,
        public aircraftName?: string,
        public aircraftType?: AircraftType,
        public aircraftReleaseDate?: Date,
        public exploitationTimeSpan?: Date/*TimeSpan*/) { }
}
